src/metatrain/cli/eval.py

```diff
@@ -79,6 +79,13 @@ def _add_eval_model_parser(subparser: argparse._SubParsersAction) -> None:
         default="output.xyz",
         help="filename of the predictions (default: %(default)s)",
     )
+    parser.add_argument(
+        "-c",
```

Comment:
Do we really want a short form for this flag? Since it is not going to be used often, I think just having the long & clearer version of the flag is good.

Reply:
I agree, will be done